CONTRIBUTING.md

```diff
@@ -101,12 +101,21 @@ $ git checkout master
 $ git pull origin master
 $ git tag vX.X.X
 ```
-- Push it to the remote repository:
-```bash
-$ git push --tag origin master
-```
 
-3. ...
+⚠️ If changes where made to the repository between your testing branch was created and the moment it was merged, you should consider building the image and testing it again. Some important changes may have been introduced, unexpectedly changing the behavior of the image that will be published to the Marketplace.
+
+3. In the [DigitalOcean Vendor Protal](https://marketplace.digitalocean.com/vendorportal), click on the title of the `MeiliSearch` Image. A form will open for a new image submission. Update the information regarding the new version in the form:
+
+- Update the `App version` (with the version number, without the starting v, so `vX.X.X` becomes `X.X.X`).
+- In the `System image` field, click on `Select system image` and select the image you have tested from the list (`MeiliSearch-v.X.X.X-Debian-X`).
+- In the `Software Included` field, update the MeiliSearch version.
+- Check the `Application summary`, `Application Description` and `Getting started instructions` fields for any inconsistent information that should be updated about MeiliSearch usage or installation.
+- In the `Reason for update` field, write "Bump MeiliSearch to vX.X.X".
+- Verify the form, and hit on `Submit`.
+
+⚠️ When the image is submitted to the Marketplace, MeiliSearch will immediately lose it's ownership. The submitted image won't appear anymore in the organization dashboard, and no further modification can be done.
+
+This will start the DigitalOcean review process. This can take a few days, and the result will be notified via email to the DigitalOcean account admins. If the image is accepted, it will be automatically published on the Marketplace. If it is rejected, an email explaining the problems will be sent to administrators.
```

Comment:
```suggestion
This will start the DigitalOcean review process. This can take a few days, and the result will be notified via email to the DigitalOcean admin account. If the image is accepted, it will be automatically published on the Marketplace. If it is rejected, an email explaining the problems will be sent to administrators.
```

Reply:
(not sure, it just sounds weird for me)